spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
Can you protect Simplecov.start with  `if ENV['COV']` or `unless ENV['NO_COV']` or similar?
I'm usually using mutant with this gem, mutant runs hundreds of times the specs and it is a PITA to run coverage each time